data.sql

```diff
@@ -145,7 +145,83 @@ VALUES
         22
     );
 
+/* Insert data into owners table */
+INSERT INTO
+    owners (full_name, age)
+VALUES
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
+
+/* Insert data into species table */
+INSERT INTO
+    species (name)
+VALUES
+    ('Pokemon'),
+    ('Digimon');
+
+/* Modify inserted animals so it includes the species_id value */
+/* SELECT * FROM animals
+WHERE name LIKE '%mon'; */
```

Comment:
- [ ] Dear @lmunozm1702, Your code is clean and to keep it that way lets not leave commented code on the codebase, remember if you ever wanna see the code you have wrote once you can use your commit history to go back.

Reply:
Hi @Tadesse-Alemayehu it's done!.

Reply:
Good job @lmunozm1702